docs/modules/ROOT/pages/commands-list.adoc

```diff
@@ -0,0 +1,176 @@
+= Comprehensive List of Commands
```

Comment:
There's already a section for this on the commands page.

The term "commands" for documentation commands might be ambiguous with command line commands.

This could be something like "Documentation Command Reference" but the navtitle would have to be shorter like "Documentation Commands" or something,